README.md

```diff
@@ -7,7 +7,7 @@ You're working on your Tunr app and you encounter this error. What does it mean
 ![Rails error](http://i.imgur.com/9NR7XNT.png)  
 
 ```text
-Your answer...
+This means that there is an issue with whatever view is trying to be rendered. I would first go to app/views to see if there was a naming error or if I needed to create the appropriate view.
 ```
 
```

Comment:
in this case, probably missing a file called `apps/views/artists/index.html.erb`